go/vt/sqlparser/ast_format.go

```diff
@@ -2656,8 +2656,12 @@ func (node *Count) Format(buf *TrackedBuffer) {
 }
 
 func (node *CountStar) Format(buf *TrackedBuffer) {
-	buf.astPrintf(node, "%s(", node.AggrName())
-	buf.WriteString("*)")
+	if node.Name != "" {
```

Comment:
@GuptaManan100 I'd like a 👍 on this fix because it wasn't in the original patch. Without the fallback to `node.AggrName()` many tests were returning `(*)` instead of `COUNT(*)`

Reply:
I'm no mr Gupta, but you can have my 👍 
I was surprised we didn't do this earlier

Reply:
Yep, this would work too I think